fuzzers/005-tilegrid/generate_full.py

```diff
@@ -300,8 +306,6 @@ def create_segment_for_int_lr(
             segtype=database[tile]["type"],
             verbose=verbose,
         )
-    else:
```

Comment:
Why was this assertion removed?

Reply:
Mainly because the else branch did not affect the behaviour given that no segment is added if the if condition is not met.

Reply:
We discussed this and I think more fundamental rework is needed on this. I'm changing title to WIP, please change back once ready for review again